### ISSUE TITLE
Whirlwind error when activating

### ISSUE THREAD

**Issue by ApostasyUK** [Baledin/FlyCasual#7]
** Fly Casual Release Version**
2.5.20230908.03

**Describe the bug**
When activating Whirlwind when the special ability triggers to receive focus tokens for being in enemy arcs and choosing to always do this, the error in the screenshot below appears. Closing this and tapping on Whirlwind again results in the same message and more focus tokens being applied but it is not possible to move on. 

**To Reproduce**

**Expected behavior**

**Screenshots**
![Screenshot_20240216-125538](https://github.com/Baledin/FlyCasual/assets/160242190/d1cde394-6aaa-4d9a-b189-85b54a3dc542)

**Platform/Device (please complete the following information):**
 - OS: Android
 - Version 14

**Additional context**

**Comment by Baledin** [comment]
I'm having difficulty replicating this situation. Further, your GitHub profile is private it appears which means I can't look at a larger version of the image so I'm barely able to make it out with my old eyes. Can you try to replicate it and provide a link to the image from a file sharing site instead (or make your GitHub profile public)?

Sorry for the extra asks, I'll see what I can do with this one--I fly Whisper as well :D

**Comment by ApostasyUK** [comment]
Sorry, I've created this account solely to report the issue as you were good enough to ensure there was a 64-bit release when I asked, so I'm not sure how to go about making the profile public. I can find instructions on making repositories public but I haven't created any so it seems unlikely that is what I need to do.

Other than that, I have tried accessing the screenshot via a private tab while not logged in without any issue and I was then able to zoom, but happy to make any change necessary to assist with this.

I made the report from my mobile but I will provide more detail when I am at my PC. The issue has cropped up both times I've tried to play the list so it wasn't a one-off but I'll play it again and see if I can be more certain of the steps to replicate.

Thank you for your continued efforts.
